libs/src/services/creatorNode/index.js

```diff
@@ -19,6 +19,23 @@ class CreatorNode {
    */
   static getSecondaries (endpoints) { return endpoints ? endpoints.split(',').slice(1) : [] }
 
+  /**
+   * Pulls off the user's clock value from a creator node endpoint and the user's wallet address.
+   * @param {string} endpoint creator node endpoint
+   * @param {string} wallet user wallet address
+   */
+  static async getClockValue (endpoint, wallet) {
+    try {
+      return (await axios({
+        url: `/users/clock_status/${wallet}`,
+        method: 'get',
+        baseURL: endpoint
+      })).data.clockValue
+    } catch (err) {
+      throw new Error(`Failed to get clock value for endpoint: ${endpoint} and wallet: ${wallet} with ${err}`)
```

Comment:
I think you want to consider how this method will be used. Should we throw an error or should we return null + just log out the error?

Reply:
I think we should discuss further on this.

I think returning a struct with empty params for the fields used might be better so that the script doesn't error when calling over thousands of calls, but I am not really sure about the direction here as well.